modules/atomic-widgets/widgets/atomic-svg.php

```diff
@@ -0,0 +1,92 @@
+<?php
+namespace Elementor\Modules\AtomicWidgets\Widgets;
+
+use Elementor\Modules\AtomicWidgets\Controls\Types\Image_Control;
+use Elementor\Modules\AtomicWidgets\PropTypes\Classes_Prop_Type;
+use Elementor\Modules\AtomicWidgets\PropTypes\Svg_Prop_Type;
+use Elementor\Modules\AtomicWidgets\PropTypes\Primitives\String_Prop_Type;
+use Elementor\Modules\AtomicWidgets\Controls\Section;
+use Elementor\Modules\AtomicWidgets\Base\Atomic_Widget_Base;
+use Elementor\Core\Utils\Svg\Svg_Sanitizer;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+class Atomic_Svg extends Atomic_Widget_Base {
+	public function get_name() {
+		return 'a-svg';
+	}
+
+	public function get_title() {
+		return esc_html__( 'Atomic SVG', 'elementor' );
+	}
+
+	public function get_icon() {
+		return 'eicon-shape';
+	}
+
+	protected function render() {
+		$settings = $this->get_atomic_settings();
+
+		if ( ! isset( $settings['svg'] ) ) {
+			return;
+		}
+
+		$svg = $settings['svg'];
+
+		$svg = new \WP_HTML_Tag_Processor( $svg );
+
+		if ( $svg->next_tag() ) {
+			$svg->set_attribute( 'fill', $settings['color'] ?? 'currentColor' );
+			$svg->set_attribute( 'width', $settings['width'] ?? '100%' );
+			$svg->set_attribute( 'height', $settings['height'] ?? '100%' );
+			$svg->set_attribute( 'min-width', $settings['min-width'] ?? '100%' );
+			$svg->set_attribute( 'min-height', $settings['min-height'] ?? '100%' );
+			$svg->set_attribute( 'max-width', $settings['max-width'] ?? '100%' );
+			$svg->set_attribute( 'max-height', $settings['max-height'] ?? '100%' );
+
+			if ( isset( $computed_style['-webkit-text-stroke'] ) ) {
+				$stroke_parts = explode( ' ', $computed_style['-webkit-text-stroke'] );
+				if ( count( $stroke_parts ) === 2 ) {
+					$svg->set_attribute( 'stroke-width', $stroke_parts[0] ?? '0px' );
+					$svg->set_attribute( 'stroke', $stroke_parts[1] ?? 'currentColor' );
+				}
+			}
+
+			$svg->add_class( $settings['classes'] ?? '' );
+		}
+
+		$valid_svg = ( new Svg_Sanitizer() )->sanitize( $svg->get_updated_html() );
+
```

Comment:
This will be caught by our external tool, please specify (in the comment) why you added phpcs:ignore